save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -377,17 +377,26 @@ internal class OrganizationController(
             // Find and remove all corresponding data to the current git repository from DB and file system storage
             val git = gitService.getByOrganizationAndUrl(organization, url)
             val testSuitesSources = testSuitesSourceService.findByGit(git)
+            // List of test suites for removing data from storage at next step
+            val testSuitesList: MutableList<TestSuite?> = mutableListOf()
```

Comment:
actually, we don't need `testSuite`, we need `TestSuitesSource` and version from `TestSuite`

Reply:
We need `organization.name`, `TestSuitesSource.name` and `TestSuite.version`, I think it more convenient to pass `testSuite` instead of this tuple, since it contain all necessary fields